app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+def test_zero_age() -> None:
+    assert get_human_age(0, 0) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(0, 0)}"
+    assert get_human_age(0, 10) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(0, 10)}"
+    assert get_human_age(10, 0) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(10, 0)}"
+
+
+def test_first_year_age() -> None:
+    assert get_human_age(1, 1) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(1, 1)}"
+    assert get_human_age(14, 14) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(14, 14)}"
+    assert get_human_age(15, 15) == [1, 1],\
+        f"Expected [1, 1], but got {get_human_age(15, 15)}"
```

Comment:
In `test_first_year_age`, the expected output for the first two assertions is [0, 0], which seems to be repeated. Ensure that each test case is unique and necessary.